src/index.ts

```diff
@@ -183,6 +186,8 @@ export class FileSystem {
     );
   }
 
+  // remotePath: path of folder to transfer in container;
+  // localPath: path to save the transferred folder on local machine
```

Comment:
/**
*Transfer folderfrom container to host machine.
*@param remotePath: folder path in contianer workspace.
*@param localPath: folder path in host machine.
*/

Reply:
Fixed.